src/index/CompressedRelation.cpp

```diff
@@ -146,10 +146,14 @@ CompressedRelationReader::asyncParallelBlockGenerator(
       RuntimeParameters().get<"lazy-index-scan-num-threads">();
 
   ad_utility::Timer popTimer{ad_utility::timer::Timer::InitialStatus::Started};
+  auto storeTime = [&details, &popTimer]() {
+    details.blockingTime_ =
+        std::chrono::duration_cast<std::chrono::milliseconds>(popTimer.value());
+  };
```

Comment:
This also gets much simpler again once we can directly get the milliseconds from the timer (it is the "typical" resolution for all of our use cases anywayl.